otherlibs/stdlib_stable/dune

```diff
@@ -24,7 +24,9 @@
    -safe-string
    -strict-formats
    -extension-universe
-   stable))
+   beta
+   -extension
+   small_numbers))
```

Comment:
This change looks suspect.  I don't think we should be changing to the `beta` extension universe, and I don't think `-extension small_numbers` should be necessary (extension-universe stable should get you it).